README.md

```diff
@@ -99,3 +99,36 @@ To instrument the .NET BindingRedirect sample application run:
 ```sh
 sampleAppTargetFramework=net472 sampleApp=BindingRedirect ./poc.sh
 ```
+
+## Handling of Assembly version Conflicts
```

Comment:
shouldn't this section live in  [docs/USAGE.md#troubleshooting](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/poc-otel-sdk/docs/USAGE.md#troubleshooting) ?